presto-druid/src/test/java/io/prestosql/plugin/druid/TestDruidSimpleQueries.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.druid;
+
+import io.prestosql.tests.AbstractTestQueryFramework;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+@Test
+public class TestDruidSimpleQueries
+        extends AbstractTestQueryFramework
```

Comment:
you are missing a lot of tests here. I think the minimum is to run `AbstractTestIntegrationSmokeTest` and `AbstractTestDistributedQueries`.

Also add separate travis job for DRUID, (see `.travis.yml`).